include/ubiquity_motor/motor_hardware.h

```diff
@@ -91,7 +96,7 @@ class MotorHardware : public hardware_interface::RobotHW {
     virtual ~MotorHardware();
     void clearCommands();
     void readInputs();
-    void writeSpeeds();
+    void writeSpeeds(bool zeroSpeeds);
```

Comment:
I don't like this function signature, when you are calling it with `writeSpeeds(true)` it is completely unclear what the true or false means. In fact it looks more like true would mean actually sending the speeds.

I would make a seperate function called `zeroSpeeds` or `stopRobot` where it is very obvious from the call site what is happening.